File: app/src/main/java/com/neocaptainnemo/cv/model/ContactsResponse.java
package com.neocaptainnemo.cv.model;

import com.google.firebase.database.IgnoreExtraProperties;
import com.google.firebase.database.PropertyName;

import java.util.Map;

@IgnoreExtraProperties
public class ContactsResponse {

    @PropertyName("contacts")
    public Contacts contacts;

    @PropertyName("resources")
    public Map<String, Map<String, String>> resources;


}

File: app/src/main/java/com/neocaptainnemo/cv/model/ContactsResponse.kt
package com.neocaptainnemo.cv.model

import com.google.firebase.database.IgnoreExtraProperties
import com.google.firebase.database.PropertyName

@IgnoreExtraProperties
class ContactsResponse {

    @PropertyName("projects")
    var contacts: Contacts? = null

    @PropertyName("resources")
    var resources: HashMap<String, Map<String, String>> = hashMapOf()


}
